vllm/model_executor/models/deepseek_v2.py

```diff
@@ -486,14 +489,10 @@ def __init__(
     ) -> None:
         super().__init__()
         config = vllm_config.model_config.hf_config
-        cache_config = vllm_config.cache_config
         quant_config = vllm_config.quant_config
         self.config = config
         self.quant_config = quant_config
-        self.model = DeepseekV2Model(config,
-                                     cache_config,
-                                     quant_config,
-                                     prefix="model")
+        self.model = DeepseekV2Model(vllm_config=vllm_config, prefix="model")
```

Comment:
We should use `maybe_prefix` to join the prefixes together. Same for the other models where `prefix` was previously given.

Reply:
fixed in [14267bf](https://github.com/vllm-project/vllm/pull/10205/commits/14267bf8b20f0d6a47714ba06e2a3b58673d22f8)

Reply:
nice catch!